pkg/cleaner/jobcontroller.go

```diff
@@ -104,8 +104,8 @@ func NewCleanupJob(bd *v1alpha1.BlockDevice, volMode VolumeMode, namespace strin
 	podTemplate.Spec = podSpec
 
 	labels := map[string]string{
-		controller.KubernetesLabelPrefix: nodeName,
-		BDLabel:                          bd.Name,
+		controller.KubernetesHostNameLabel: nodeName,
```

Comment:
hmm. should we delete the old label? To catch if there are other places where we might be using it?

Reply:
We are not deleting the old label. Both `kubernetes.io/hostname` and `blockdevice` label will be present on the job